sdk/python/express_relay/svm/generated/express_relay/instructions/set_splits.py

```diff
@@ -26,7 +26,7 @@ def set_splits(
     remaining_accounts: typing.Optional[typing.List[AccountMeta]] = None,
 ) -> Instruction:
     keys: list[AccountMeta] = [
-        AccountMeta(pubkey=accounts["admin"], is_signer=True, is_writable=True),
+        AccountMeta(pubkey=accounts["admin"], is_signer=True, is_writable=False),
```

Comment:
ditto here, can set to False for all the other instructions where you remove `mut`

Reply:
done